contents/newsletter/talk-to-users.md

```diff
@@ -0,0 +1,106 @@
+---
+title: An engineer's guide to talking to users
+date: 2024-04-18
+author:
+  - ian-vanagas
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/newsletter/beyond-10x-engineer/super-hog.png
+featuredImageType: full
+tags:
+  - Engineering
+  - Product engineers
+crosspost:
+  - Product engineers
+  - Blog
+---
+
+Talk to users? Isn't that a job for founders, product managers, user researchers, and salespeople? Why do would an engineer ever want or need to do it?
```

Comment:
maybe to avoid the three ??? something more like

Why would an engineer ever need to talk to users? That's a job for founders, product managers, user researchers, and salespeople. Right? Wrong.

In his book...